README.md

```diff
@@ -208,6 +213,183 @@ try {
 
 <a name="help"/>
 
+<a name="in-process-linking"/>
+
+## Trace in-process asynchronous execution
+
+You can use the SDK to trace asynchronous in-process code execution. This might be useful if Dynatrace does not support the threading framework or specific asynchronous libraries. In-process-linking will only be used to link other services (Database-, Webrequests, ...) between thread or queuing boundaries, where currently no support is available.
+
+To link asynchronous execution, you need to receive a InProcessLink, where async execution forks:
+
+```Java
+InProcessLink inProcessLink = OneAgentSDK.createInProcessLink();
+```
+
+The provided inProcessLink must not serialized and can only be used inside the process, where it has been created. It must be used to start tracing where the async execution takes place:
+
+```Java
+InProcessLinkTracer tracer = oneAgentSDK.traceInProcessLink(inProcessLink);
+
+tracer.start();
+try {
+	// do the asynchronous job
+} catch (Exception e) {
+	tracer.error(e);
+} finally {
+	tracer.end();
+}
+```
+
+<a name="messaging"/>
+
+## Trace messages
+
+You can use the SDK to trace messages sent or received via messaging system. When tracing messages, we distinct between:
+* sending a message
+* receiving a message
+* processing a received message
+
+To trace an outgoing message, the code looks straight forward compared to other tracers:
```

Comment:
the code looks straight forward compared to other tracers => should this indicated that code for other traces looks complicated/strange?
I would expect some statement/description regarding the dynatrace tag to be added. Currently this is only in sample code.